src/pages/LocationsPage.spec.js

```diff
@@ -0,0 +1,26 @@
+import { render, screen } from '@testing-library/react'
+import LocationsPage from './LocationsPage'
+
+describe('LocationsPage', () => {
+  it('renders a list of all locations with an image and description', () => {
+    render(
+      <LocationsPage
+        anyLocation={[
+          {
+            decription: 'Garten mit Haus',
+            image: 'image',
+          },
+          {
+            description: 'Garten mit Partyraum',
+            image: 'another image',
+          },
+        ]}
+      />
+    )
+
+    const description = screen.getAllByText('Garten')
+    expect(description).toHaveLength()
```

Comment:
What length do you expect it to have? Is this working just like that?

Reply:
I just copied some testing-code to be honest. I think i don't need to test the length of anything. The testing is still on my to do list. :) It's not really working and it says it couldn't find anything with "Garten" - strange for me but I need to work on the test anyways and maybe rewrite it. Need some help with that.

Reply:
It won't be able to find it because your component doesn't accept any props. If you implement it like I suggested and pass `locations` to your `LocationsPage` then this should work. Well, not the `toHaveLength()` but with `toBeInTheDocument()`.